src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -16,8 +16,10 @@
 import seedu.address.commons.core.LogsCenter;
 import seedu.address.model.crew.Crew;
 import seedu.address.model.flight.Flight;
+import seedu.address.model.flight.exceptions.FlightNotFoundException;
```

Comment:
Using a shared `NotFoundException` shall be good enough. No need to do anything for now, we can do refactor later.